lib/src/material_color_picker.dart

```diff
@@ -3,22 +3,22 @@ import 'package:flutter_material_color_picker/src/colors.dart';
 import 'package:flutter/material.dart';
 
 class MaterialColorPicker extends StatefulWidget {
-  final Color selectedColor;
-  final ValueChanged<Color> onColorChange;
-  final ValueChanged<ColorSwatch> onMainColorChange;
-  final List<ColorSwatch> colors;
+  final Color? selectedColor;
+  final ValueChanged<Color>? onColorChange;
+  final ValueChanged<ColorSwatch?>? onMainColorChange;
```

Comment:
The type should be `ValueChanged<ColorSwatch>?`, we never call it with a null value.